modules/library/coverage/src/main/java/org/geotools/coverage/processing/operation/Crop.java

```diff
@@ -445,7 +445,7 @@ public Coverage doOperation(ParameterValueGroup parameters, Hints hints) {
 					cropRoi, 
 					roiTolerance,
 					forceMosaic,
-                    (hints instanceof Hints) ? (Hints) hints: new Hints(hints),
+                    (hints instanceof Hints) ? hints: new Hints(hints),
```

Comment:
API must of been changed, probably had the possibility of being a Map previously.